app/views/physical_ticket/show.pdf.prawn

```diff
@@ -0,0 +1,60 @@
+prawn_document(filename: @file_name, page_layout: :portrait, :page_size =>'A4' ) do |pdf|
+  # Vertical Layout
+  top = pdf.bounds.top
+  bottom = pdf.bounds.bottom
+  left = pdf.bounds.left
+  right = pdf.bounds.right
+  mid_vertical = (pdf.bounds.top-pdf.bounds.bottom)/2
+  mid_horizontal = (pdf.bounds.right-pdf.bounds.left)/2
+  x = 0
+
+  pdf.move_down mid_vertical
+  pdf.dash(2, :space => 1)
+  pdf.stroke_horizontal_rule
+  pdf.stroke_vertical_line pdf.bounds.top, pdf.bounds.bottom, :at => mid_horizontal
+  pdf.move_up mid_vertical
+  pdf.draw_text "TICKET HOLDER", :at => [x,pdf.cursor-30], :size => 17
+  pdf.dash(2, :space => 0)
+  pdf.stroke_rectangle  [x, pdf.cursor-50], 230, 150
+  pdf.move_down 80
+  pdf.draw_text "NAME", :at => [x+10,pdf.cursor], :size => 13
+  pdf.fill_color "808080"
+  pdf.draw_text "#{@user.name}", :at => [x+10,pdf.cursor-25], size: 20
+  pdf.fill_color "000000"
+  pdf.draw_text "EMAIL", :at => [x+10,pdf.cursor-50], :size => 13
+  pdf.fill_color "808080"
+  pdf.draw_text "#{@user.email}", :at => [x+10,pdf.cursor-75], size: 20
+  pdf.fill_color "000000"
+  pdf.move_up 20
+  if @conference.picture?
+    pdf.image "#{Rails.root}/public#{@conference.picture_url}", :at => [mid_horizontal+30, pdf.cursor], :height => 70
```

Comment:
no, because then you will change the dimension of the images. I think the best option would be to set only the biggest of the two. I mean, if the width is higher, then set the height. If not, set the height. This way it won't overflow.

Please when doing it, add here screenshots with a long and tall image, so we can see how it looks like :wink: